skyvern/webeye/scraper/domUtils.js

```diff
@@ -714,6 +714,10 @@ function buildTreeFromBody() {
     }
   }
   function processElement(element, parentId) {
+    if (element === null) {
+      return;
```

Comment:
@LawyZheng 

let's console.log something here so that we can monitor issues like this from web console in the future

Reply:
Sure, though there is no valuable info to log here, since the element is null and parentId might be null.